pluto_build/sql/qaqc_outlier.sql

```diff
@@ -0,0 +1,50 @@
+-- -v VERSION=$VERSION
+DROP TABLE IF EXISTS qaqc_outlier;
+CREATE TABLE IF NOT EXISTS qaqc_outlier(
+    v character varying,
+    CONDO character varying,
+    MAPPED character varying,
+    outlier character varying
+);
+
+DELETE FROM qaqc_outlier
+WHERE v = :'VERSION'
+AND CONDO::boolean = :CONDO
+AND MAPPED::boolean = :MAPPED;
+
+INSERT INTO qaqc_outlier (
+select :'VERSION' as v, 	 
+       :CONDO as condo,
+       :MAPPED as mapped,
+       jsonb_agg(t) as outlier
+from (
+	select jsonb_agg(json_build_object('bbl', tmp.bbl, 'unitsres',tmp.unitsres, 'resarea', tmp.resarea,
+                              'res_unit_ratio', tmp.res_unit_ratio)) as values, 'unitsres_resarea' as field
+	from (SELECT bbl, unitsres, resarea, resarea::INT/unitsres::INT as res_unit_ratio
+          FROM (SELECT a.* 
```

Comment:
I am adding another subquery for the condo and mapped condition here, as I am not quite sure about how to really concatenate the input argument with other conditions in the where clause. I have tried several methods but seems they don't work properly. 

If anyone can come up with a nicer solution to make the code simpler, please point out for me. I feel the code is a bit too long right now, which is not really necessary.